src/index.ts

```diff
@@ -551,13 +735,13 @@ export function zipFolder(srcFolder: string, zipFilePath: string) {
 
 export interface RokuDeployOptions {
     /**
-     * A full path to the folder where the zip package should be placed
+     * A full path to the folder where the zip/pkg package should be placed
      * @default "./out"
      */
     outDir?: string;
     /**
-     * The name the zip file should be given. 
-     * @default "roku-deploy.zip"
+     * Basename the zip/pkg file should be given.
```

Comment:
Change to something like "Base filename the zip/pkg file should be given (excluding the extension)"